content/4_Gold/TopoSort.mdx

```diff
@@ -654,7 +690,7 @@ public class Main {
 
 		PrintWriter pw = new PrintWriter(System.out);
 
-		ArrayDeque<Integer> answer = new ArrayDeque<>();  // Acts as a Stack
+		ArrayDeque<Integer> answer = new ArrayDeque<>();  //Acts as a Stack
```

Comment:
why are the above comments spaced by 1 but not this and line 720

Reply:
the comments i see (lines 620, 627, 644, 661) all don't have spaces, which one do you see that has spaces? i might have missed something

Reply:
the solution you added

Reply:
he means that the c++ sol had "comments spaced by 1 but not this"

Reply:
Oh. Most comments in the java sol didn't have spaces, so I changed this one to be consistent since I didn't want to change the code too much. I think just being consistent within the code is fine, but I can add spaces to the comments in the java code if you want.

Reply:
it's up to you I was just curious :)

Reply:
but my personal preference would to be add spaces